src/sqlancer/cnosdb/CnosDBExpectedError.java

```diff
@@ -0,0 +1,80 @@
+        ERRORS.add("requested character too large for encoding.");
+        ERRORS.add("Can not find compatible types to compare Boolean with [Utf8].");
+        ERRORS.add("Cannot create filter with non-boolean predicate 'APPROXDISTINCT");
+        ERRORS.add("HAVING clause references non-aggregate values:");
+        ERRORS.add("Cannot create filter with non-boolean predicate");
+        ERRORS.add("negative substring length not allowed");
+        ERRORS.add("The function Sum does not support inputs of type Boolean.");
+        ERRORS.add("The function Avg does not support inputs of type Boolean.");
+        ERRORS.add("Percentile value must be between 0.0 and 1.0 inclusive");
+        ERRORS.add("Date part '' not supported");
+        ERRORS.add("Min/Max accumulator not implemented for type Boolean.");
+        ERRORS.add("meta need get_series_id_by_filter");
+        ERRORS.add("Arrow: Cast error:");
+        ERRORS.add("Arrow error: Cast error:");
+        ERRORS.add("Datafusion: Execution error: Arrow error: External error: Arrow error: Cast error:");
+        ERRORS.add("Arrow error: Divide by zero error");
+        ERRORS.add("desired percentile argument must be float literal");
+        ERRORS.add("Unsupported CAST from Int32 to Timestamp(Nanosecond, None)");
+        ERRORS.add("Execution error: Date part");
+        ERRORS.add("Physical plan does not support logical expression MIN(Boolean");
+        ERRORS.add("The percentile argument for ApproxPercentileCont must be Float64, not Int64");
+        ERRORS.add("The percentile argument for ApproxPercentileContWithWeight must be Float64, not Int64.");
+        ERRORS.add("Data type UInt64 not supported for binary operation '#' on dyn arrays.");
+        ERRORS.add("Arrow: Divide by zero error");
+        ERRORS.add("The function ApproxPercentileCont does not support inputs of type Null.");
+        ERRORS.add("can't be evaluated because there isn't a common type to coerce the types to");
+        ERRORS.add("This was likely caused by a bug in DataFusion's code and we would welcome that you file an bug");
+        ERRORS.add("The function ApproxMedian does not support inputs of type Null.");
+        ERRORS.add("null character not permitted.");
+        ERRORS.add("The percentile argument for ApproxPercentileCont must be Float64, not Null.");
+        ERRORS.add("This feature is not implemented");
+        ERRORS.add("The function Avg does not support inputs of type Null.");
+        ERRORS.add("Coercion from [Utf8, Timestamp(Nanosecond, Some(\\\"+00:00\\\"))]");
+        ERRORS.add(
+                "Coercion from [Utf8, Float64, Utf8] to the signature OneOf([Exact([Utf8, Int64]), Exact([LargeUtf8, Int64]), Exact([Utf8, Int64, Utf8]), Exact([LargeUtf8, Int64, Utf8]), Exact([Utf8, Int64, LargeUtf8]), Exact([LargeUtf8, Int64, LargeUtf8])]) failed.");
+        ERRORS.add("Coercion from");
+    }
+
+    public static ExpectedErrors expectedErrors() {
```

Comment:
I think that this method should be called `getExpectedExpressionErrors` since all the errors seem specific to expressions? This is something that could be done in a future PR.